packages/mock/src/delay.ts

```diff
@@ -5,7 +5,7 @@ export const getDelay = (
   options?: GlobalMockOptions,
 ): GlobalMockOptions['delay'] => {
   const overrideDelay =
-    typeof override?.mock?.delay === 'number'
+    override?.mock?.delay !== undefined
```

Comment:
A bit unsure about this, but it feels like override always should override when present, and not only when it's overridden by a number?

Reply:
Good question. Not sure why this decision was made?

Reply:
Maybe it was missed when support for functions was added?